src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public void doWork() {
+        System.out.println("Excavator started its work.");
```

Comment:
The message in 'doWork' method should be more informative. It should indicate that it is an Excavator that has started working. Consider including the type of machine in the message.